.github/workflows/wasm_backend_tests_ci.yml

```diff
@@ -94,26 +94,26 @@ jobs:
       - name: Build examples
         run: VTEST_ONLY=wasm ./v build-examples
 
-  wasm-backend-windows:
-    runs-on: windows-2022
-    if: github.event_name != 'push' || github.event.ref == 'refs/heads/master' || github.event.repository.full_name != 'vlang/v'
-    timeout-minutes: 121
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Build V
-        run: .\make.bat -msvc
-      - name: Symlink V
-        run: .\v.exe symlink -githubci
-
-      - name: Install binaryen as build dependency for the V WASM backend
-        run: v cmd/tools/install_binaryen.vsh
-
-      - name: Build the V WASM backend
-        run: v -cc msvc -showcc -v cmd/tools/builders/wasm_builder.v
-
-      - name: Test the WASM backend
-        run: v -stats test vlib/v/gen/wasm/tests/
-
-      - name: Build examples
-        run: $env:VTEST_ONLY='wasm'; v build-examples
+#  wasm-backend-windows:
```

Comment:
I thought you had a command to install wasmer for Windows.

Reply:
I am currently making sure that I can at least get Ubuntu and MacOS working, then I'll add windows back in.